wordpress-mobile/gutenberg-mobile
# Extract duplicate e2e test setup to jest config

See Gutenberg PR for more information: https://github.com/WordPress/gutenberg/pull/27334

To test:

- CI: Full E2E Test suite should pass
- Locally: `npm run test:e2e:ios:local` and `npm run test:e2e:android:local` should work as before.

PR submission checklist:
